packages/EasyStandard/src/Sniffs/Exceptions/ThrowExceptionMessageSniff.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyStandard\Sniffs\Exceptions;
+
+use Nette\Utils\Strings;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+
+final class ThrowExceptionMessageSniff implements Sniff
+{
+    /**
+     * Processes this test, when one of its tokens is encountered.
+     *
+     * @param \PHP_CodeSniffer\Files\File $phpcsFile
+     * @param int $stackPtr
+     */
+    public function process(File $phpcsFile, $stackPtr): void
+    {
+        $tokens = $phpcsFile->getTokens();
+
+        $nextTokenPosition = TokenHelper::findNextEffective($phpcsFile, $stackPtr + 1);
+        $nextToken = $tokens[$nextTokenPosition];
+
+        if ($nextToken['code'] !== \T_NEW) {
+            return;
+        }
+
+        $openParenthesisToken = TokenHelper::findNext($phpcsFile, [\T_OPEN_PARENTHESIS], $stackPtr);
+
+        $messageTokenPosition = TokenHelper::findNextEffective($phpcsFile, $openParenthesisToken + 1);
+        $messageToken = $tokens[$messageTokenPosition];
+
+        if ($messageToken['code'] === \T_VARIABLE) {
+            return;
+        }
+
+        if ($messageToken['code'] === \T_CLOSE_PARENTHESIS) {
+            return;
+        }
+
+        if (Strings::startsWith($messageToken['content'], "'exceptions.") === false) {
```

Comment:
Hmm.. Ok I think that this sniff is really opinionated but I can see some good in it 😄 

My only real issue is that all translations are not necessarily gonna start with `exceptions.` that depends on the app. I think that this sniff should accept an optional array of prefixes to test against, if the `$messageToken['content']` doesn't start with at least one of them then the sniff should fail.

You can default this list of prefixes to `['exceptions.']` if you want I don't mind.

Reply:
Done.